source/framework/11.x/validation.php

```diff
@@ -120,28 +120,30 @@
         'symbols'       => 'The :attribute field must contain at least one symbol.',
         'uncompromised' => 'The given :attribute has appeared in a data leak. Please choose a different :attribute.',
     ],
-    'present'              => 'The :attribute field must be present.',
-    'present_if'           => 'The :attribute field must be present when :other is :value.',
-    'present_unless'       => 'The :attribute field must be present unless :other is :value.',
-    'present_with'         => 'The :attribute field must be present when :values is present.',
-    'present_with_all'     => 'The :attribute field must be present when :values are present.',
-    'prohibited'           => 'The :attribute field is prohibited.',
-    'prohibited_if'        => 'The :attribute field is prohibited when :other is :value.',
-    'prohibited_unless'    => 'The :attribute field is prohibited unless :other is in :values.',
-    'prohibits'            => 'The :attribute field prohibits :other from being present.',
-    'regex'                => 'The :attribute field format is invalid.',
-    'required'             => 'The :attribute field is required.',
-    'required_array_keys'  => 'The :attribute field must contain entries for: :values.',
-    'required_if'          => 'The :attribute field is required when :other is :value.',
-    'required_if_accepted' => 'The :attribute field is required when :other is accepted.',
-    'required_if_declined' => 'The :attribute field is required when :other is declined.',
-    'required_unless'      => 'The :attribute field is required unless :other is in :values.',
-    'required_with'        => 'The :attribute field is required when :values is present.',
-    'required_with_all'    => 'The :attribute field is required when :values are present.',
-    'required_without'     => 'The :attribute field is required when :values is not present.',
-    'required_without_all' => 'The :attribute field is required when none of :values are present.',
-    'same'                 => 'The :attribute field must match :other.',
-    'size'                 => [
+    'present'                => 'The :attribute field must be present.',
+    'present_if'             => 'The :attribute field must be present when :other is :value.',
+    'present_unless'         => 'The :attribute field must be present unless :other is :value.',
+    'present_with'           => 'The :attribute field must be present when :values is present.',
+    'present_with_all'       => 'The :attribute field must be present when :values are present.',
+    'prohibited'             => 'The :attribute field is prohibited.',
+    'prohibited_if'          => 'The :attribute field is prohibited when :other is :value.',
+    'prohibited_if_accepted' => 'The :attribute field is prohibited when :other is accepted.',
+    'prohibited_if_declined' => 'The :attribute field is prohibited when :other is declined.',
```

Comment:
@Laravel-Lang/laravel-lang, hi!

I missed that point. Laravel 11.43 added two new keys, `prohibited_if_accepted` and `prohibited_if_declined`.

PR: https://github.com/laravel/framework/pull/54608
Release: [laravel/framework#v11.43.0](https://github.com/laravel/framework/releases/tag/v11.43.0)